lib/interfaces/gql-module-options.interface.ts

```diff
@@ -1,19 +1,26 @@
 import { Type } from '@nestjs/common';
 import { ModuleMetadata } from '@nestjs/common/interfaces';
 import {
-  Config,
-  IResolverValidationOptions,
-  ServerRegistration,
+  Config as ConfigExpress,
+  IResolverValidationOptions as IResolverValidationOptionsExpress,
+  ServerRegistration as ServerRegistrationExpress,
 } from 'apollo-server-express';
```

Comment:
What if someone is using `apollo-server-fastify` and doesn't have `apollo-server-express` installed locally? Isn't the build going to show errors because of the missing dependencies (references to non-existing libs in the declaration files `.d.ts`)?

Reply:
@kamilmysliwiec yup this I wasn't sure how to solve, any ideas how we could fix this?

Reply:
this is something that we cannot fix easily. Very likely the only existing solution is to actually duplicate these interfaces in our repo.

Reply:
@kamilmysliwiec I removed the `apollo-server-express` and `apollo-server-fastify` and added the interfaces within the file itself.